bump_version

```diff
@@ -54,16 +54,6 @@ else
     new_version_deb="${new_version}-1"
 fi
 
-# Set the Dockerfile web version to the specified new_version
-old_version="$(
-    grep "JELLYFIN_WEB_VERSION=" Dockerfile \
-        | sed -E 's/ARG JELLYFIN_WEB_VERSION=v([0-9\.]+[-a-z0-9]*)/\1/'
-)"
-echo $old_version
-
-old_version_sed="$( sed 's/\./\\./g' <<<"${old_version}" )" # Escape the '.' chars
-sed -i "s/${old_version_sed}/${new_version}/g" Dockerfile*
-
```

Comment:
I don't think this section is needed any more because the Docker files now specify `JELLYFIN_WEB_VERSION=master` instead of a specific version, but I'm not sure about this

Reply:
Leave this change out, this file is under review in a separate PR: https://github.com/jellyfin/jellyfin/pull/2656

Reply:
Removed it